.github/dependabot.yml

```
@@ -0,0 +1,16 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
```

No reason to not also make this daily? :)
```suggestion
    interval: daily
```

---

Depends how often Jinja updates, but sure